src/app/models/profile.ts

```diff
@@ -6,8 +6,11 @@ export class Profile {
     bio?: string;
     username?: string;
     url?: string;
-    emails?: Email[];
-    primaryEmail?: Email;
+    emails?: string[];
```

Comment:
I disagree, the client side data model should cleanly represent what the UX is. You then use the service to adapt that on to your server side endpoints.